pkg/volume/aws_ebs/aws_ebs.go

```diff
@@ -291,6 +290,10 @@ func (b *awsElasticBlockStoreBuilder) SetUpAt(dir string) error {
 		return err
 	}
 
+	if !b.readOnly {
```

Comment:
I don't quite follow... Why do we need special handling for fsGroup == 0. Please note that fsGroup is an *int64 for which the unset case is fsGroup == nil